samples/multiplatform/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.61'
+}
+repositories {
+    mavenCentral()
+}
+group 'com.example'
+version '0.0.1'
+
+apply plugin: 'maven-publish'
+
+kotlin {
+    jvm()
+    js {
+        browser {
+        }
+        nodejs {
+        }
+    }
+    // For ARM, should be changed to iosArm32 or iosArm64
+    // For Linux, should be changed to e.g. linuxX64
+    // For MacOS, should be changed to e.g. macosX64
+    // For Windows, should be changed to e.g. mingwX64
+    mingwX64("windows")
```

Comment:
That's for the native target, Atrium does not yet support it and thus you can remove it. However, I would like to see an Android target. See
https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#android-support

Reply:
the android target is a bit mere complicated so might be best to create an issue for it

Reply:
fine with me